### DISCUSSION TITLE
Community contributions

### DISCUSSION THREAD

**Discussion by matthewdennett** [infobloxopen/infoblox-ansible#150] [General]
Hi All, 

I just wanted to ask what the expectations are around community contributions to this project? 

I have been extensively using this module over the past few months and quickly found some major pain points with some functionality not being implemented. The two notable pain points I ran into was member assignment to a network and no support for managing network ranges amongst a few others. @Medalibi and I both have PRs open to fix the issues we have but there seems to be very little or no interest from here to review/approve our changes. 

I have a few more changes I would like to submit, but when a PR is taking the better part of a month the motivation to keep providing improvements to this project is slowly dwindling. Is it expected that contributions will move this slowly?

**Comment by ranjishmp** [comment]
Hi @matthewdennett 

One of our key maintainer left the org and that's why you are seeing the slowness. We are trying to get other folks upto speed. So may need some time to get settle down with these

Definitely we are looking for community contributions. Will review those PRs soon.

Thanks,
Ranjish

**Reply by matthewdennett** [reply]
Thanks for getting back to me @ranjishmp . 

I'm happy for any and all comments on the PR's I haven't submitted many in the public space before.

**Comment by asAlmas** [comment]
Hello @matthewdennett

We have had a key maintainer leave the organization, which is why you are seeing the slowness right now. There is a lot of work to be done in order to get other folks up to speed. This may take some time for me to adjust to these changes, so I might need some time
The community is definitely welcome to contribute to this project[.](https://wonapk.com/free-fire-bundles/) It will be my pleasure to review those PRs in the near future.

**Reply by philipsd6** [reply]
Good to hear @asAlmas! I hope things improve soon, but in the meantime, perhaps someone can at least approve my PR #157 workflow? As a first time contributor, my PR isn't going through the workflow and if it could at least do that I can ensure the PR is ready for review.

**Reply by ranjishmp** [reply]
@philipsd6 I have requested @hemanthKa677 to review the same

**Comment by matthewdennett** [comment]
I feel this needs to be echoed again. It's getting close to 2 years later and almost nothing from the community seems to have moved on this repo. 

There are a couple changes that have been put though and they seem to be get a snappy turn around. What are we missing from our contributions that don't warrant any feedback?

**Reply by SanjeevManurkar** [reply]
We have been working on supporting our automation efforts for Ansible (and Terraform). We have had 2 versions released last year for Ansible and we are planning on addressing few PR/Issues this year as well. Thank you for your continued patience on this.

**Reply by matthewdennett** [reply]
Hi @SanjeevManurkar, It was great to see 2 version released last year. However, would it not make more sense to try and action PR as they come in, keep people encouraged to commit more the the project instead of the delay we are seeing now and the frenzy to do things right before the release?

**Reply by SanjeevManurkar** [reply]
Ideally, yes. But please note that we do not have dedicate resources working on just one provider like Ansible. Our resources shuffle between multiple providers and work based on the priorities set for them. We take the PRs/Issues submitted seriously, prioritize those, and commit to subset of them based on resource availability, so that there is no frenzy before the release timelines.
